src/adapter/src/coord/metrics.rs

```diff
@@ -23,6 +24,10 @@ pub struct Metrics {
 impl Metrics {
     pub fn register_with(registry: &MetricsRegistry) -> Self {
         Self {
+            envd_start_time: registry.register(metric!(
+                name: "mz_start_time_environmentd",
```

Comment:
Seems weird for a metric named `mz_start_time_environmentd` to live in the coordinator. I feel like it should either live in the `mz_environmentd` crate, or be named `mz_start_time_adapter`.